### DISCUSSION TITLE
`vertico-buffer-mode` + `display-buffer-below-selected`

### DISCUSSION THREAD

**Discussion by hron** [minad/vertico#506] [Q&A]
I have a big display and I would like vertico to display itself below selected window occupying the same width as the window. This is how I'm trying to achieve such behavior:

```elisp
(use-package! vertico
  :config
  (setq vertico-buffer-display-action '(display-buffer-below-selected (side . bottom)))
  (vertico-buffer-mode +1))
```

Unfortunately, the window is always occupies the full frame's width. Is there a solution to achieve this? Check the screenshot. Basically, I want vertico to be displayed the same way `*RE-builder*` is displayed. For `*RE-builder*` it is achievable by configuring `display-buffer-alist` with something like the following, but it doesn't work for vertico...

```elisp
(setq display-buffer-alist
      '(
        ("\\*RE-Builder\\*"
          (display-buffer-below-selected (side . bottom)))
))
```

![image](https://github.com/user-attachments/assets/351aa953-5ac8-4619-b216-5cd7679d6d77)

**Comment by minad** [comment]
Hi, I suspect that the `display-buffer-below-selected` action doesn't work correctly with `vertico-buffer-mode`. Therefore I also haven't it listed in the `defcustom`. According to the Emacs help, this is not a new display action, so no new incompatibility.
